openstudiocore/src/model/RenderingColor_Impl.hpp

```diff
@@ -40,13 +40,8 @@ class RenderingColor;
 
 namespace detail {
 
-  /** RenderingColor_Impl is a ResourceObject_Impl that is the implementation class for RenderingColor.*/
-  class MODEL_API RenderingColor_Impl : public ResourceObject_Impl {
-
-
-
-
-
+  /** RenderingColor_Impl is a ModelObject_Impl that is the implementation class for RenderingColor.*/
+  class MODEL_API RenderingColor_Impl : public ModelObject_Impl {
```

Comment:
@jmarrec RenderingColor should implement the parent call here.

Reply:
Ok, I'll override ModelObject::parent() and implement it then. thanks for the catch